### PR TITLE
Integrate with AZP logging

### PR DESCRIPTION
**Pull request by staticfloat**

### CODE REVIEW

**Comment by giordano** [comment]
Maybe remove the license in a commit?

**Comment by staticfloat** [comment]
There are lots of `@info` messages that should have been caught I think?

File: .ci/Manifest.toml [hunk 1]
```diff
@@ -245,9 +250,9 @@ version = "1.1.0"
 
 [[RegistryTools]]
 deps = ["AutoHashEquals", "LibGit2", "Pkg", "UUIDs"]
-git-tree-sha1 = "94cf95fd962cc0594da4d676ed66b553f84ec246"
+git-tree-sha1 = "3dfa671318ac8af835cc621b08fcdcc28a4aee67"
 uuid = "d1eb7eb1-105f-429d-abf5-b0f65cb9e2c4"
-version = "1.3.4"
```
Comment (giordano) [inline .ci/Manifest.toml:250]: This shouldn't be downgraded, see https://github.com/JuliaPackaging/Yggdrasil/pull/777

Comment (staticfloat) [reply .ci/Manifest.toml:250]: Good catch; the resolver did that to me.  I guess we should force a compat entry here.

Comment (staticfloat) [reply .ci/Manifest.toml:250]: sigh, old registries for the fail

**Comment by staticfloat** [comment]
INTERESTING.  So it looks like it's working! 🎉

I kind of wish the warnings were visible from here; I'm going to try making the license issue an `@error` instead of a `@warn` and see what it looks like here.

**Comment by staticfloat** [comment]
Alright, this isn't perfect (e.g. we still get a green checkmark no matter what kind of messages we spew out, so we have to manually click through to see the warnings/errors) but it's still very useful for us to be able to click in and at a glance see what has been done.

**Comment by staticfloat** [comment]
Ah, I guess I'll still merge this once I merge the PR in BB.

**Pull request merged**